admin/app/components/solidus_admin/ui/table/component.yml

```diff
@@ -3,3 +3,9 @@ en:
   rows_selected: 'selected'
   select_all: 'Select all'
   select_row: 'Select row'
+  search: Search
+  close: Close
+  filter: Filter
+  placeholders:
+    search_key:
+      name_or_variants_including_master_sku_cont: Search by Name or Sku
```

Comment:
How about including a line in the preference comment that clarifies the need to create a new key with the same value if changed?

Reply:
That makes sense 👍  I've incorporated a comment and assigned a default value for the translation.